Week2/akif.js

```diff
@@ -0,0 +1,108 @@
+// Part 1 Hello World
+console.log('Hello World'); // English
+console.log('Merhaba Dunya'); // Turkish
+console.log('Hallo Wereld'); // Dutch
+console.log('مرحبا بالعالم'); // Arabic
+
+// Part 2 Error  -----------------------------------------
+console.log("I'm awesome");
+
+// Part 3 Console.log(x)   --------------------------------
+let x;
+console.log('the value of my variable x will be: numberOfTheStudents');
+console.log(x);
+x = 16;
+console.log('the value of my variable x will be: numberOfTheStudents');
+console.log(x);
+
+// Part 4  String y  ---------------------------------------
+let y = 'I love you JavaScript';
+console.log('the value of my variable y will be: feelingAboutJavaScript');
+console.log(y);
+y = 'I will learn you';
+console.log('the value of my variable y will be: myPlanAboutJavaScript');
+console.log(y);
+
+// Part 5  Round a number  ---------------------------------
+let z = 7.25;
```

Comment:
Can use `const` here instead of `let`, isn't it? Because we don't reassign `z` or `a` or `b`.